apps/studio/src/assets/styles/app/vendor/datepicker/var.scss

```diff
@@ -0,0 +1,35 @@
+$namespace: 'mx' !default;
+
+$default-color: #73879c !default;
+$primary-color: #1284e7 !default;
```

Comment:
Does this all work in light mode, solarized, and solarized dark?

Reply:
I tried it in light and dark mode and it looked pretty good.